unittests/CppInterOp/VariableReflectionTest.cpp

```diff
@@ -131,6 +133,9 @@ TEST(VariableReflectionTest, DatamembersWithAnonymousStructOrUnion) {
             ((intptr_t) & (k3.c)) - ((intptr_t) & (k3.num)));
   EXPECT_EQ(Cpp::GetVariableOffset(datamembers_klass3[4]),
             ((intptr_t) & (k3.num2)) - ((intptr_t) & (k3.num)));
+#ifdef _WIN32
+#pragma warning(default : 4201)
```

Comment:
Maybe worth a comment what warning that is.

Reply:
Comment added.